### PR TITLE
Rename spec device posture

### PR DESCRIPTION
**Pull request by darktears**
This needs to be done in sync with a repository rename as well.

***
<a href="https://pr-preview.s3.amazonaws.com/darktears/screen-fold/pull/60.html" title="Last updated on Mar 9, 2021, 1:49 PM UTC (0bb5524)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/screen-fold/60/29d96ea...darktears:0bb5524.html" title="Last updated on Mar 9, 2021, 1:49 PM UTC (0bb5524)">Diff</a>

### CODE REVIEW

**Comment by darktears** [comment]
It's for reference only, I'll rebase it when the screen fold angle removal lands.

**Comment by darktears** [comment]
@anssiko @kenchris @diekus

File: index.html [hunk 1]
```diff
@@ -3,15 +3,15 @@
   <head>
     <meta charset='utf-8'>
     <title>
-      The Screen Fold API
+      The Device Posture API
```
Comment (anssiko) [inline index.html]: I'd drop "The" for consistency with the established naming conventions.

**Review by anssiko — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (anssiko) [reply index.html]:
```suggestion
      Device Posture API
```

Comment (anssiko) [reply index.html]: @darktears I fixed that for you in https://github.com/w3c/screen-fold/pull/60/commits/e9b8bfc6ba8727f969ef9e6102698c11b7f82f14

**Comment by anssiko** [comment]
@xfq in order to coordinate this change, please rename this repo from "screen-fold" to "device-posture", set the redirects and merge this PR once done. (Travis CI fail seems unrelated.)

**Pull request merged**